backend/src/schema/types/type/NOTIFIED.gql

```diff
@@ -1,14 +1,14 @@
 type NOTIFIED {
   id: ID!
-  from: NotificationSource
-  to: User
   createdAt: String!
   updatedAt: String!
-  read: Boolean
-  reason: NotificationReason
+  read: Boolean!
+  reason: NotificationReason!
+  from: NotificationSource!
+  to: User!
 }
 
-union NotificationSource = Post | Comment
+union NotificationSource = Post | Comment | Report
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-23T19:04:31Z" title="Thursday, January 23rd 2020, 8:04:31 pm +01:00">Jan 23, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,14 +1,14 @@
 type NOTIFIED {
   id: ID!
-  from: NotificationSource
-  to: User
   createdAt: String!
   updatedAt: String!
-  read: Boolean
-  reason: NotificationReason
+  read: Boolean!
+  reason: NotificationReason!
+  from: NotificationSource!
+  to: User!
 }
 
-union NotificationSource = Post | Comment
+union NotificationSource = Post | Comment | Report
```


our union is growing!!

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-01-24T16:54:34Z" title="Friday, January 24th 2020, 5:54:34 pm +01:00">Jan 24, 2020</time>_

---


Yeah 😄